docs/expose-applications-using-loadbalancer-type-service.md

```diff
@@ -70,17 +70,52 @@ Request Body:
 
 ### Service annotations
 - loadbalancer.openstack.org/floating-network-id
+
+  The floating network id which will allocate floating ip for loadbalancer. This annotation works when the value of `service.beta.kubernetes.io/openstack-internal-load-balancer` is false.
 - loadbalancer.openstack.org/floating-subnet
+
+  A floating network can have several subnets. This annotation is one of subnets' name.
 - loadbalancer.openstack.org/floating-subnet-id
+
+  This annotation is one of subnets' id in a floating network.
 - loadbalancer.openstack.org/class
+
+  The name which predefined floating networks and subnets in config file.
```

Comment:
Suggestion: `The name of a preconfigured class in the config file.`

Please also describe the relationship between `loadbalancer.openstack.org/class` and the above options.

Reply:
Thanks for your suggestion.